app/assets/stylesheets/custom.css.scss

```diff
@@ -151,3 +157,34 @@ aside {
   margin-top: 15px;
 }
 
+/* forms */
+
+input, textarea, select, .uneditable-input {
+  border: 1px solid #bbb;
+  width: 100%;
+  margin-bottom: 15px;
+  @include box_sizing;
```

Comment:
Name of mixin `box_sizing` should be written in all lowercase letters with hyphens instead of underscores